src/WheelsAndGoods.Application/ServiceCollectionExtensions.cs

```diff
@@ -25,6 +25,7 @@ public static class ServiceCollectionExtensions
 
         services.AddSingleton<IBlobService, BlobService>();
         services.AddSingleton<IEmailSenderService, EmailSenderService>();
+        services.AddScoped<IOrdersService, OrdersService>();
```

Comment:
Please move this one to 25 line and make space. You can see that singletones are separated, please follow the style.